packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -56,6 +57,43 @@ keywords: [menu, dropdown, action menu]
 
 The MenuTrigger accepts exactly two children: the element which triggers the opening of the Menu and the Menu itself. The trigger element must be the first child passed into the MenuTrigger and should support press events.
 
+## Long press
+By default, a MenuTrigger's Menu is opened by pressing the trigger element or activating it via the <Keyboard>Space</Keyboard> and <Keyboard>Enter</Keyboard> keys. However, there may be cases in which your trigger element
+should perform a separate default action on press and should only display the Menu when long pressed. This behavior can be set by providing "longPress" to the `trigger` prop. When done, the Menu will only be opened upon
+pressing and holding the trigger element or by using the <Keyboard>Option</Keyboard> (<Keyboard>Alt</Keyboard> on Windows) + <Keyboard>Down Arrow</Keyboard> keys while focusing the trigger element.
+
+The example below illustrates how one would setup a MenuTrigger to have long press behavior.
+
+```tsx example
```

Comment:
Maybe a better example would be one that swaps out the tool for whatever is selected? Essentially a demo of exactly how photoshop uses this component?